<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setPortDialog</class>
 <widget class="QDialog" name="setPortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置端口</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>224</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>选项</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="typeComboBox">
      <item>
       <property name="text">
        <string>vxlan</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>key</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="keyLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="ipLineEdit"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
